src/components/song/song.service.ts

```diff
@@ -158,35 +160,36 @@ export class SongService {
     ];
 
     try {
-      const query = this.db
+      const result = await this.db
         .query()
         .call(matchRequestingUser, session)
         .match([
-          node('rootUser', 'User', {
+          node('root', 'User', {
             active: true,
             id: this.config.rootAdmin.id,
           }),
         ])
         .call(createBaseNode, ['Song', 'Producible'], secureProps, {
           owningOrgId: session.owningOrgId,
         })
-        .create([...this.permission('range', 'node')])
-        .call(addUserToSG, 'rootUser', 'adminSG')
-        .call(addUserToSG, 'rootUser', 'readerSG')
-        .return('node.id as id');
+        .create([...this.permission('scriptureReferences', 'node')])
+        .return('node.id as id')
+        .first();
 
-      const result = await query.first();
       if (!result) {
         throw new ServerException('failed to create a song');
       }
 
-      const id = result.id;
-
-      // add root admin to new song as an admin
-      await this.db.addRootAdminToBaseNodeAsAdmin(id, 'Song');
```

Comment:
This is the only one of the 4 that had this. Is this missing on all 4 now or is it not needed?

Reply:
It is duplicated in `createBaseNode` function. So not needed.